### DISCUSSION TITLE
processing non `RedrawRequested` event after the main event loop: AboutToWait

### DISCUSSION THREAD

**Discussion by Zrs000** [bevyengine/bevy#12934] [Q&A]
When I run the iOS example on iPhone, the console repeatedly shows: processing non `RedrawRequested` event after the main event loop: AboutToWait, how can I fix this?

![image](https://github.com/bevyengine/bevy/assets/49590420/201f59b7-84cf-44d3-af83-8b5638fd4b3a)

**Comment by ActuallyHappening** [comment]
Exactly the same, is there some way to disable this specific warning? Looking for something to chuck in the `EnvFilter`

**Comment by ActuallyHappening** [comment]
I am still debugging this issue, note also that you can set `NO_COLOR` env variable to get rid of those useless ANSI escape codes https://github.com/bevyengine/bevy/pull/13984

I am resorting to implementing a filter layer

**Reply by ActuallyHappening** [reply]
I am still getting this issue, with many noisy logs being printed every frame.
In the end I simply forked this from the point where the crates.io release was build, see here: https://github.com/ActuallyHappening/winit/tree/remove-noisy-logs

**Comment by Piefayth** [comment]
see also

https://github.com/rust-windowing/winit/issues/3714
